tests/test_pyproject_pep660.py

```diff
@@ -58,6 +58,7 @@ def test_pep660_pip_isolated(isolated, isolate):
     assert value == "4.0"
 
     location = isolated.execute("import simplest; print(*simplest.__path__)")
+    print(location)
     # First path is from the python source
```

Comment:
Oh, the path was not printed truncated, it's just assert that was showing it truncated. Anyway it seems `platlib` was constructed with inconsistent case-sensitivity. Might as well just make those asserts case-insesitive if they are on windows

Reply:
Do you know where the little "l" lib is coming from? It would be nice to fix it to match the big "L" Lib, ideally, as case normalization is complicated. macOS is also usually case insensitive, but not entirely. If the files exist, we can use os.path.samefile, I think.

Reply:
About `platlib`:
```python
        self.platlib = Path(
            self.execute("import sysconfig; print(sysconfig.get_path('platlib'))")
        )
```
Probably was patched, but it was never backported